pages/projusers/publications.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Scientific publications using Scikit-HEP
+permalink: /publications
+nav_order: 20
+parent: Who uses Scikit-HEP?
+---
+
+
+# Scientific publications using Scikit-HEP {#publication_users}
```

Comment:
We keep track of `pyhf`'s use in analyses and citation count here: https://scikit-hep.org/pyhf/citations.html#use-in-publications and then for IRIS-HEP I maintain a similar page as well: https://iris-hep.org/projects/pyhf.html

Do we want to duplicate this again? Or should we just link a subset of them here?

Reply:
These duplications are a pain but are almost unavoidable. Personally I'm going to put as much as possible related to my business on the website, as that's then centralised. But there are cases where it may not be ideal, and I have no strong feeling nor a perfect solution.

The page I'm adding here organises by areas rather than packages, but you could argue the other way around. Some dynamic selection would be super cool but that's not for tomorrow, I'm sure. Hence this pragmatic approach.

For this "version 0" I think I would link to your page https://scikit-hep.org/pyhf/citations.html#use-in-publications for now until we have info related to the other packages, and we then discuss a better solution? I really think it is important that our site advertises the impact Scikit-HEP is having.

On presentations: we actually have https://scikit-hep.org/resources that is by-now totally out-of-date. Updating it is also on my to-do list but of course it would be easier if every package maintainer would add for their packages every now and then.

Is this acceptable? I believe yes since you and Henry are happy for this "version 0" to go in.

Reply:
> For this "version 0" I think I would link to your page https://scikit-hep.org/pyhf/citations.html#use-in-publications for now until we have info related to the other packages, and we then discuss a better solution? I really think it is important that our site advertises the impact Scikit-HEP is having

:+1: SGTM. Thanks @eduardo-rodrigues!

Reply:
Cool. Done with the latest commit. Will be merging shortly today.